src/server.ts

```diff
@@ -152,18 +156,21 @@ async function announce(
 	try {
 		await indexNewTorrents();
 		const result = await checkNewCandidateMatch(candidate);
-		if (!result) {
-			res.writeHead(204);
-			res.end();
-			return;
+		const isOk = result === InjectionResult.SUCCESS || result === SaveResult.SAVED;
+		if (!isOk) {
+			if (result === InjectionResult.TORRENT_NOT_COMPLETE) {
+				res.writeHead(202);
```

Comment:
why 202?

Reply:
It looked like the best one for this case.

"Accepted response status code indicates that the request has been accepted for processing, but the processing has not been completed; in fact, processing may not have started yet."

If you wanted to use a different one, I'm all up for it.